src/change_processor.rs

```diff
@@ -283,6 +283,11 @@ fn compute_and_apply_changes(tree: &mut RojoTree, vfs: &Vfs, id: Ref) -> Option<
                 let snapshot = match snapshot_from_vfs(&metadata.context, &vfs, &path) {
                     Ok(Some(snapshot)) => snapshot,
                     Ok(None) => {
+                        // If the file that didn't return an instance is a model.json file
+                        // no return is expected as empty/whitespace mode.json files are ignored
+                        if let Some(_) = match_file_name(path, ".model.json") {
+                            return None;
+                        }
```

Comment:
This does not feel like the correct solution here. I'd rather not special-case `.model.json` files and instead come back to this problem in another PR.

Alternatively, should we make `.model.json` files return a dummy instance, like an empty `Folder`? I think that returning `None` may cause Rojo to skip watching the file for changes, but I'm not sure.